packages/react/src/components/RuleBuilder/utils.js

```diff
@@ -132,7 +132,7 @@ export const getRuleByPath = (arr, path) => {
  */
 export const generateRule = () => {
   return {
-    id: Math.random().toString(36).substring(2, 12),
+    id: Math.random().toString(36).substring(2, 12).padStart(10, '0'),
```

Comment:
This seems unnecessary just to get a test to pass. Maybe its better to remove the length check in the test instead unless you're something specific is happening that makes it need 10 numbers

Reply:
@sstone2423 Good point. Used that approach in newest push.